### PR TITLE
New dev env for Steve Grundell

### PR DESCRIPTION
**Pull request by natarajaya**
This PR adds common code to create dev env for Steve Grundell.

### CODE REVIEW

File: common/live/prd/infra/dev/sgrundell/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    paths = ["../zone"]
+  }
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+
+project_name    = "dev-sgrundell"
```
Comment (stepanstipl) [inline common/live/prd/infra/dev/sgrundell/terraform.tfvars]: Is `sgrundell` Steve's username?

Comment (natarajaya) [reply common/live/prd/infra/dev/sgrundell/terraform.tfvars]: Yes, his @rtf.org account.

Comment (stepanstipl) [reply common/live/prd/infra/dev/sgrundell/terraform.tfvars]: People's email addresses often don't match usernames they use: `sergey@` vs `natarajaya`, `javi@` vs `jhernandez`...

Comment (natarajaya) [reply common/live/prd/infra/dev/sgrundell/terraform.tfvars]: Yeah, I double checked with Steve and he is fine with this.

**Pull request merged**